.gitignore

```diff
@@ -1 +1,133 @@
+# Include your project-specific ignores in this file
+# Read about how to use .gitignore: https://help.github.com/articles/ignoring-files
+# Useful .gitignore templates: https://github.com/github/gitignore
+
+# Git files
+*.diff
+*.orig
+*.rej
+
+# Numerous always-ignore extensions
+pids
+*.pid
+*.seed
+*.lock
+*.err
+*.tmp
+*.log*
+
+# Vim
+[._]*.s[a-w][a-z]
+[._]s[a-w][a-z]
+*.un~
+Session.vim
+.netrwhist
+*~
+
+# Text Editors
+.swo
+*.swp
+*.vi
+
+# Numerous always-ignore directories
+logs
+tmp
+
+# Windows files and directories
+Thumbs.db
+ehthumbs.db
+ehthumbs_vista.db
+Image.db
+Video.db
+TVThumb.db
+musicThumbs.db
+thumbcache_*.db
+
+# Mac files and directories
+.DS_Store
+.AppleDouble
+.LSOverride
+.Spotlight-V100
+.Trashes
+.AppleDB
+.AppleDesktop
+Network Trash Folder
+Temporary Items
+.apdisk
+
+# Thumbnails
+._*
+
+# IntelliJ IDEA and other products
+*.iml
+.idea
+release
+
+# VSCode metadata
+.vscode
+
+# Sublime
+*.sublime-project
+*.sublime-workspace
+.sublimelinterrc
+
+# Eclipse
+.project
+.classpath
+.settings
+
+# gitkeep
+!.gitkeep
+
+# Directory for instrumented libs generated by `jscoverage/JSCover`
+lib-cov
+
+# Coverage directory used by tools like `istanbul`, `phpunit/php-code-coverage` and etc.
+coverage
+
+# nyc test coverage
+.nyc_output
+
+# node-waf configuration
+.lock-wscript
+
+# Compiled binary addons (http://nodejs.org/api/addons.html)
+build/Release
+
+# Dependency directories
+bower_components
+.bower-cache
+.bower-registry
+.bower-tmp
 node_modules
+jspm_packages
+
+# Optional npm cache directory
+.npm
+
+# Optional REPL history
+.node_repl_history
+
+# Ignore minified files
+*.min.*
+
+# Grunt intermediate storage (http://gruntjs.com/creating-plugins#storing-task-files)
+.grunt
+
+# Python
+*.pyc
+
+# Caches
+.cache
+.cache-loader
+.eslintcache
+.eclintercache
+.stylelintcache
+.sass-cache
+.phpcscache
+
+# npm
+npm-shrinkwrap.json
+!package-lock.json
+
+
```

Comment:
Does anything from this list appears in your project directory? This is trash IMO.

Reply:
@TrySound it is standard .gitignore, `log`, `.idea`, `*.imp` and other trash

Reply:
Also `.eslintcache` from `eslint`, i think better keep this list as is, there a lot of edge cases for difference os/editors/etc